index.html

```diff
@@ -9,42 +9,42 @@
 </head>
 <body>
     <appbar id="appbar">
-        <img src="./assets/App Bar - Compact - Dark - iOS.svg" alt="">
+        <img src="./assets/App Bar - Compact - Dark - iOS.svg" alt="Application bar">
     </appbar>
 
     <header id="header">
-        <item_nav id="item_nav">
-            <fram4>
+        <div id="item_nav">
+            <div>
                 <portf id="portf"><p>portfolio</p></portf>
-            </fram4>
-            <fram3>
+            </div>
+            <div>
                 <about id="about"><p>About</p></about>
-            </fram3>
-            <fram5>
+            </div>
+            <div>
                 <contact id="contact"><p>Contact</p></contact>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly use valid HTML5 tags. There is no tag called `portf` or `about` or `contact`...etc 👍kindly refer to [this](https://www.w3schools.com/TAGS/default.asp)

Reply:
The previous owner did not tell me to change elements in pre approved earlier projects, I changed those elements now.